api/server/router/local/container.go

```diff
@@ -30,11 +30,12 @@ func (s *router) getContainersJSON(ctx context.Context, w http.ResponseWriter, r
 	}
 
 	config := &daemon.ContainersConfig{
-		All:     httputils.BoolValue(r, "all"),
-		Size:    httputils.BoolValue(r, "size"),
-		Since:   r.Form.Get("since"),
-		Before:  r.Form.Get("before"),
-		Filters: r.Form.Get("filters"),
+		All:         httputils.BoolValue(r, "all"),
+		Size:        httputils.BoolValue(r, "size"),
+		Since:       r.Form.Get("since"),
+		Before:      r.Form.Get("before"),
+		Filters:     r.Form.Get("filters"),
+		IgnoreLinks: httputils.BoolValue(r, "ignoreLinks"),
```

Comment:
I actually implemented this at the beginning with trunc, but I think having a dedicated flag is better because we're not using it for anything else but ignoring the links.